demo-private.html

```diff
@@ -44,12 +45,19 @@
 </head>
 <body unresolved>
 
+<google-signin
+  clientId="1054047045356-j8pgqgls9vdef3rl09hapoicumbte0bo.apps.googleusercontent.com"
```

Comment:
Would you mind mentioning this in the docs and adding an example that shows needing an external `<google-signin>` now. That's an API change that users will need to know about.